dts/arm/nxp/nxp_mcxn94x_common.dtsi

```diff
@@ -567,6 +567,22 @@
 		interrupts = <57 0>;
 		status = "disabled";
 	};
+
+	wwdt0: watchdog@16000 {
+		compatible = "nxp,lpc-wwdt";
+		reg = <0x16000 0x1000>;
+		interrupts = <152 0>;
+		status = "disabled";
+		clk-divider = <1>;
+	};
```

Comment:
the commit adding these nodes should be after the commit where they are actually supported

also, the commit title should have spaces after colons

and the name of the soc is mcxn94x not mcxn994x